source/js/buyers-guide/components/creep-vote/creep-vote.jsx

```diff
@@ -176,63 +146,15 @@ export default class CreepVote extends React.Component {
                 onChange={(value) => this.setCreepiness(value)}
               />
             </div>
-            <div className="col-12 col-md-6 mt-5 mt-md-0">
-              <div className="mb-4 text-center">
-                <h3 className="h5-heading mb-2">
-                  {`How likely are you to ${this.buyOrUse} it?`}
-                </h3>
-              </div>
-              <div className="text-center">
-                <div
-                  className="btn-group btn-group-toggle mt-3 mt-md-5"
-                  data-toggle="buttons"
-                >
-                  <label htmlFor="likely">
-                    <input
-                      type="radio"
-                      name="wouldbuy"
-                      id="likely"
-                      autoComplete="off"
-                    />
-                    <span
-                      className={likelyClasses}
-                      onClick={() => this.setConfidence(true)}
-                      onKeyPress={(evt) => this.setConfidence(true, evt.key)}
-                      tabIndex="0"
-                      role="button"
-                    >
-                      Likely
-                    </span>
-                  </label>
-                  <label htmlFor="unlikely">
-                    <input
-                      type="radio"
-                      name="wouldbuy"
-                      id="unlikely"
-                      autoComplete="off"
-                    />
-                    <span
-                      className={unlikelyClasses}
-                      onClick={() => this.setConfidence(false)}
-                      onKeyPress={(evt) => this.setConfidence(false, evt.key)}
-                      tabIndex="0"
-                      role="button"
-                    >
-                      Not likely
-                    </span>
-                  </label>
-                </div>
-              </div>
-            </div>
           </div>
           <div className="row">
             <div className="col-12 text-center">
               <button
                 id="creep-vote-btn"
                 type="submit"
-                className="btn btn-secondary mb-2"
+                className="btn btn-pop mb-2"
               >
-                Vote & See Results
+                Vote & see results
```

Comment:
How is i18n handled in JavaScript?

Reply:
I _think_ iirc we need to use `getText(..)`here. Dont quote me on that, but I believe I've seen that before.

Reply:
JSX localization is a [long standing problem](https://github.com/mozilla/foundation.mozilla.org/pull/4181) that should be revisited later.